### DISCUSSION TITLE
PostgreSQL Integration label not working for the form input in database export/import configuration customization

### DISCUSSION THREAD

**Discussion by krivulcik** [ravendb/ravendb#14616] [Q&A] [answered]
When I open the form for database export or import (http://live-test.ravendb.net/studio/index.html#databases/tasks/exportDatabase?&database=test), open the `Advanced` part, enable `Customize Configuration and Ongoing Tasks`, clicking the label `PostgreSQL Integration` doesn't activate the switch next to it. The label has `for="includePostgreSqlIntegration_3"`, but the input has `id="includePostgreSqlIntegration3"` (without the underscore).

![image](https://user-images.githubusercontent.com/7997055/180787505-b8ef1eee-d67b-4045-bfa8-5e99c6a8d7ad.png)

**Comment by Danielle9897** [comment, marked as answer]
Opened issue:
https://issues.hibernatingrhinos.com/issue/RavenDB-19009/ExportImport-View-Fix-PostgreSQL-toggle
